be/src/util/mem_info.cpp

```diff
@@ -63,6 +63,17 @@ int64_t MemInfo::_s_sys_mem_available_warning_water_mark = 0;
 int64_t MemInfo::_s_process_minor_gc_size = -1;
 int64_t MemInfo::_s_process_full_gc_size = -1;
 
+static std::unordered_map<std::string, int64_t> jemalloc_metrics;
```

Comment:
The metrics map is not atomic or protected by lock. Is there a chance to read wrong value?

Reply:
Fixed, the set value is no longer repeated every time a request is made